### PR TITLE
Gradient Cards

### PR DESCRIPTION
**Pull request by Siggi005**
### New Stylesheet Submissions:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders a compiled CSS file?

### CODE REVIEW

File: src/_data/styles/gradientcards.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Gradient Cards",
+  "author": "Bente Hinkenhuis",
+  "stylesheet": "https://raw.githubusercontent.com/Siggi005/Stylestage/master/style.css",
```
Comment (5t3ph) [inline src/_data/styles/gradientcards.json:4]:
```suggestion
  "stylesheet": "https://raw.githubusercontent.com/Siggi005/Stylestage/master/style.css"
```

Extra comma invalidates JSON :)

**Comment by 5t3ph** [comment]
Hi @Siggi005! this is a great start! There are quite a few contrast errors, and those will need to be resolved in order to be approved.

Here's your preview: https://deploy-preview-30--moderncss-styles.netlify.app/styles/gradient-cards/

You can look up browser extension tools like WAVE or aXe to help show you where there are contrast errors.

I also noticed that the nav needs to wrap for mobile, but otherwise the layout elements look good!

**Comment by 5t3ph** [comment]
Hi @Siggi005 I just wanted to check in with you and see if you needed any help with these issues?

I just learned today that firefox will help show you contrast errors with gradients which may help you out! https://twitter.com/5t3ph/status/1289579892439482379

**Comment by Siggi005** [comment]
Hi Stephanie,
I think I can work it out, thanks for the offer! I had an active sports week and was therefore not able to work on it yet ☹
Actually, I was using the contrast info in Firefox already, even for this Project, but did neither want to include more than three shades for one of my main colors nor filter the ones in the background… but I will decide soon!

**Comment by 5t3ph** [comment]
Hi @Siggi005  - checking in if you plan to continue? I may close this in the next few days if I do not hear back, but you are welcome to resubmit once you've had a chance to resolve the issues. Thanks!

**Comment by 5t3ph** [comment]
Hi @Siggi005 - I'm going to close this since it's been quite a month without an update. You are absolutely welcome to re-submit in the future when you have time to address the issues! Thanks so much for participating, I hope you learned some new things with what you created so far, and I hope to see your updated styles someday 😊

**Pull request closed without merging**